src/openrct2/interface/Screenshot.cpp

```diff
@@ -498,61 +498,61 @@ int32_t cmdline_for_gfxbench(const char** argv, int32_t argc)
     return 1;
 }
 
-static void ApplyOptions(const ScreenshotOptions* options, rct_viewport& viewport)
+static void ApplyOptions(const ScreenshotOptions* _options, rct_viewport& viewport)
```

Comment:
Underscore prefix are for static declarations outside functions.

Reply:
Is it better now?

Reply:
yes